viewer/packages/data-providers/src/image-360-data-providers/cdm/transformCdmAnnotations.test.ts

```diff
@@ -3,233 +3,103 @@
  */
 import { Vector3 } from 'three';
 import { transformAnnotations } from './transformCdmAnnotations';
+import { mockGetImage360WithAnnotationsFromRevisionResponse } from '../../../../../test-utilities/src/fixtures/dmsResponses/mockGetImage360WithAnnotationsFromRevisionResponse';
 import { GetImage360FromRevisionResponse } from './fetchCoreDm360AnnotationsForRevision';
 
 describe('transformAnnotations', () => {
-  it.each(['', '1.0.0'])('transforms annotation correctly with formatVersion being %s', format => {
-    const result = transformAnnotations({
-      ...mockQueryResponse,
-      items: {
-        ...mockQueryResponse.items,
-        annotations: [
-          {
-            ...mockQueryResponse.items.annotations[0],
-            properties: {
-              cdf_cdm: {
-                'Cognite360ImageAnnotation/v1': {
-                  ...mockQueryResponse.items.annotations[0].properties['cdf_cdm']['Cognite360ImageAnnotation/v1'],
-                  formatVersion: format
-                }
-              }
-            }
-          }
-        ]
-      }
-    });
+  it.each([undefined, '', 'not-a-semantic-version', '0.1.0', '1.0.0'])(
+    'transforms annotation correctly with formatVersion being %s',
+    formatVersion => {
+      const mockQueryResponse = getMockresponseWithFormatVersion(formatVersion);
+      const result = transformAnnotations(mockQueryResponse);
```

Comment:
Non-blocking: I see that this test and the next have almost identical bodies. I wonder if a more interesting separation of test cases would be 1: Check that  for all versions, the function returns the common parts of the response body (annotation, image and annotation metadata). Then, in separate tests, check that versions below and above 1.0.1 transforms the data as expected, respectively. What do you think?